README.md

```diff
@@ -56,6 +56,11 @@ Manually trigger update of postings from API. By default there is a background t
 
 By default various fields are being indexed to enable implementation of a comprehensive job search. Title, team, location & commitment are all indexed and an example query has been provided.
 
+### Features
+
+There is a RSS feed feature included in this module which allows users to access RSS feed in [MCV jobs](https://mcvuk.careerwebsite.com/r/jobs/post/batch_specs.cfm) format.   

```

Comment:
I'd suggest another section in under features that's titled "Feeds". Also, it's not an RSS feed, it's an XML feed.